### PR TITLE
Create darvapi.is-an.app

### PR DESCRIPTION
**Pull request by darvzac**
for fun

### Link to Website

Link: https://darvapi.darcoz.repl.co

### CODE REVIEW

**Review by tarampampam — CHANGES_REQUESTED** [review]
Wrong filename. Change it, please

**Comment by darvzac** [comment]
> Wrong filename. Change it, please

Done

File: domains/darvapi.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "just api project",
+  "domain": "is-an.app",
+  "subdomain": "darvapi",
+  "owner": {
+    "email": "zarconxorp@gmail.com"
+  },
+  "record": {
+    "CNAME": "270dc528-7c02-48cd-8f87-8d337bba9b9a.id.repl.co",
```
Comment (tarampampam) [inline domains/darvapi.is-an.app.json:10]:
```suggestion
    "CNAME": "270dc528-7c02-48cd-8f87-8d337bba9b9a.id.repl.co.",
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/darvapi.is-an.app.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "just api project",
+  "domain": "is-an.app",
+  "subdomain": "darvapi",
+  "owner": {
+    "email": "zarconxorp@gmail.com"
+  },
+  "record": {
+    "CNAME": "270dc528-7c02-48cd-8f87-8d337bba9b9a.id.repl.co.",
```
Comment (tarampampam) [inline domains/darvapi.is-an.app.json:10]:
```suggestion
    "CNAME": "270dc528-7c02-48cd-8f87-8d337bba9b9a.id.repl.co."
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Review by tarampampam — APPROVED** [review]

**Pull request merged**